pkg/crc/ssh/client.go

```diff
@@ -93,6 +93,11 @@ func (client *NativeClient) session() (*ssh.Session, error) {
 func (client *NativeClient) Run(command string) ([]byte, []byte, error) {
 	session, err := client.session()
 	if err != nil {
+		if client.conn != nil {
```

Comment:
This is still resetting the connection even on expected errors (for example busy loops until a given oc command succeeds) as was pointed out by Guillaume.

Reply:
@cfergeau earlier it used to but with this code shouldn't because now we are tapping the session creation error not the command execution error so it shouldn't reset connection on expected errors for oc commands.